packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -96,12 +114,6 @@ export default function Preview( { idBase, instance, isVisible } ) {
 						// TODO: This chokes when the query param is too big.
 						// Ideally, we'd render a <ServerSideRender>. Maybe by
 						// rendering one in an iframe via a portal.
-						src={ addQueryArgs( 'widgets.php', {
-							'legacy-widget-preview': {
-								idBase,
-								instance,
-							},
-						} ) }
 						onLoad={ ( event ) => {
```

Comment:
No definite recommendations, but I wonder if we still need this `onLoad` handler if the loading is now done via api fetch.

Reply:
`apiFetch` only fetches the html string, this `onLoad` handler is needed when the actual iframe content is loaded by the browser.